app/src/main/java/hm/orz/chaos114/android/carnavimodoki/activity/MainActivity.java

```diff
@@ -61,16 +62,30 @@
             MediaStore.Video.Media.TITLE
     };
 
+    public void hoge() {
+        // no-op
+    }
+
+    public void HOGEHOGE() {
+
+    }
+
     //region Activity
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+        ;
+        ;
+        ;
         setContentView(R.layout.activity_main);
+        String hoge = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa";
 
         startService(new Intent(this, MusicService.class));
 
         ButterKnife.inject(this);
 
+        hoge.getBytes();
```

Comment:
: [I18N] Found reliance on default encoding in hm.orz.chaos114.android.carnavimodoki.activity.MainActivity.onCreate(Bundle): String.getBytes()